Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/ViewBookmarkLoaderInterface.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/Serialization/SerializeContext.h>
+#include <Prefab/ViewBookmarkComponent.h>
+
+/*!
+ * ViewBookmarkIntereface
+ * Interface for saving/loading View Bookmarks.
+ */
+namespace AzToolsFramework
+{
+    class ViewBookmarkLoaderInterface
+    {
+    public:
+        AZ_RTTI(ViewBookmarkLoaderInterface, "{71E7E178-4107-4975-A6E6-1C4B005C981A}")
+
+        enum StorageMode
```

Comment:
update: If we do end up keeping this it would be better to use `enum class` here to get increased type safety